src/components/Header/Button/DarkMode/DarKMode.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useContext } from 'react';
+import { ThemeContext } from '../../../../providers/Theme/Theme.provider';

```

Comment:
I think you can remove all the `../` and start from the `src` of your folder
```suggestion
import { ThemeContext } from 'providers/Theme/Theme.provider';
```